src/scene/vcFlythrough.cpp

```diff
@@ -321,8 +322,13 @@ void vcFlythrough::HandleSceneEmbeddedUI(vcState *pProgramState)
       ImGui::EndCombo();
     }
 
-    vcIGSW_FilePicker(pProgramState, vcString::Get("flythroughExportPath"), m_exportPath, udLengthOf(m_exportPath), nullptr, 0, vcFDT_SelectDirectory, nullptr);
-    
+    // Only testing if a first frame has already been exported in this folder
+    auto testIfExistLambda = [pProgramState, this]
+        {if (!vcModals_OverwriteExistingFile(pProgramState, udTempStr("%s/%05d.%s", m_exportPath, 0, vcFlythroughExportFormats[m_selectedExportFormatIndex]), vcString::Get("flythroughExportAlreadyExists")))
```

Comment:
This should be formatted correctly, also this should be performed on export not on file pick. Exporting a second time won't prompt this.

Reply:
Moved to the Export button